doc/guides/bluetooth/overview.rst

```diff
@@ -0,0 +1,88 @@
+.. _bluetooth-overview:
+
+Overview
+########
+
+.. contents::
+    :local:
+    :depth: 2
+
+Supported Features
+******************
+
+Zephyr comes integrated with a feature-rich and highly configurable
+Bluetooth stack.
+
+* Bluetooth 5.0 compliant (ESR10)
+
+  * Bluetooth subsystem and infrastructure
+
+  * Highly configurable
+
+      * Features, buffer sizes/counts, stack sizes, etc.
+  
+  * Portable to all architectures supported by Zephyr (including big and
```

Comment:
Silly comment: The big endian support is work in progress ;)

Reply:
we'll consider it almost done ;)

Reply:
Until #13229 is merged we are a bit far from done :)